libraries/commerce/product/selectors/product.js

```diff
@@ -184,21 +184,68 @@ export const getProduct = createSelector(
   }
 );
 
+/**
+ * Retrieves the product name from the current route.
+ * @param {Object} state The current application state.
+ * @param {Object} props The component props.
+ * @return {string|null}
+ */
+export const getProductRouteName = createSelector(
```

Comment:
This selector does not belong to `commerce/product`. It belong to `common/router` and name should be `getRouteTitle`

Reply:
I did it in a slightly different way now. But this new selector is gone now.